ui-react/packages/atlasmap-core/src/services/document-management.service.ts

```diff
@@ -287,6 +290,11 @@ export class DocumentManagementService {
           inspectionType, isSource);
         break;
 
+      case DocumentType.CSV:
+        this.cfg.initializationService.initializeUserDoc(fileText, userFile, DocumentType.CSV,
```

Comment:
`await` here, I just discovered we need that for this to work

Reply:
Thanks! Good catch. Addressed in https://github.com/atlasmap/atlasmap/pull/2041